core/encoding/cli.go

```diff
@@ -77,6 +78,12 @@ func CLIFlags(envPrefix string) []cli.Flag {
 			Required: false,
 			EnvVar:   common.PrefixEnvVar(envPrefix, "PRELOAD_ENCODER"),
 		},
+		cli.StringFlag{
+			Name:     G2PowerOf2PathFlagName,
+			Usage:    "Path to G2 SRS points that are on power of 2",
```

Comment:
helpful to comment in usage that at least one of these g2 flags need to be specified; and if both are specified, which one takes precedence